demos/iCubTest/composeGui.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.7"
+
+x-icub-test: &icub-test
+  image: icubteamcode/superbuild-icubtest:master-unstable_sources
+  environment:
+    - DISPLAY=${DISPLAY}
+    - QT_X11_NO_MITSHM=1
+    - XAUTHORITY=/root/.Xauthority
+    - YARP_FORWARD_LOG_ENABLE=1
+    - YARP_ROBOT_NAME
+    - SUITE_TYPE
+    - CUSTOM_SUITE
+    - CUSTOM_FILE_PATH
+    - RESULTS_PATH
+  volumes:
+    - "/tmp/.X11-unix:/tmp/.X11-unix"
+    - "${XAUTHORITY}:/root/.Xauthority"
+    - "${HOME}/${YARP_CONF_PATH}:/root/.config/yarp"
+    - "${RESULTS_PATH:-/tmp/icub-test_results}:/projects/robotology-superbuild/src/icub-tests/suites/results:rw"
+    - "${CUSTOM_FILE_PATH:-/tmp/icub-test_results/customSuite.xml}:/projects/robotology-superbuild/src/icub-tests/suites/customSuite.xml"
```

Comment:
Same as above

Reply:
As above, but here they must be present.